yarn-project/end-to-end/src/e2e_fees/fees_test.ts

```diff
@@ -247,7 +262,7 @@ export class FeesTest {
     await this.snapshotManager.snapshot(
       'fund_alice',
       async () => {
-        await this.mintPrivate(BigInt(this.ALICE_INITIAL_BANANAS), this.aliceAddress);
+        await this.mintPrivateBananas(BigInt(this.ALICE_INITIAL_BANANAS), this.aliceAddress);
```

Comment:
:D

Reply:
It's important to be clear about what we're minting!